src/main/java/net/neoforged/neoforge/registries/DeferredRegister.java

```diff
@@ -164,7 +178,9 @@ public static DeferredRegister.Blocks createBlocks(String modid) {
      * @see #create(ResourceKey, String)
      * @see #create(ResourceLocation, String)
      * @see #createItems(String)
+     * @deprecated Use the overload with the {@link ResourceKey} parameter
```

Comment:
Just for completeness' sake, add a `{@link}` to the new method to this javadoc. (i.e., `Use {@link ...} with the ...`)

(Same comment for the `DataComponents` constructor.)